docs/filters.md

```diff
@@ -121,12 +124,15 @@ This will not work for ambiguous expressions (containing OR or NOT operators) an
 - Number ranges are written using a relational operator (.e.g `>5`, `<=10`).
 - Repo ids are written as `owner/name`
 - Sorting keys are written as a comma-separated list of possibly-negated keys. For example, `repo, ~updated, -comments`. See `sort` for more information.
+- Qualified issue ids are written as `repo_id#number` or `repo_id#number_range`.
 
 ### id
 
-*Expects a number or number range*
+*Expects a string in form of qualified issue id, number, or number range*
+
+If a qualified issue id is specified, it will match the issue from the given repo id with the given id number, or issues from the given repo id with ids in the given range.
 
-Matches the issue with the given id number, or issues with ids in the given range.
+If a number or number range is specified, it will matches the issue with the given id number, or issues with ids in the given range from the primary repo.
```

Comment:
Maybe we could be a bit more succinct. Just to merge the two paragraphs, so things aren't repeated.

```
Matches the issue with the given id number, or issues with ids in the given range. Will look in the default repository if the issue id is unqualified.
```